packages/ui/components/form/inputs/Input.tsx

```diff
@@ -12,7 +12,7 @@ import { Eye, EyeOff, X, Search } from "../../icon";
 import { HintsOrErrors } from "./HintOrErrors";
 import { Label } from "./Label";
 
-type InputProps = JSX.IntrinsicElements["input"] & { isFullWidth?: boolean; isStandaloneField?: boolean };
+type InputProps = JSX.IntrinsicElements["input"] & { isFullWidth?: boolean };
```

Comment:
This an extra, was receiving error on console, it looks like it doesn't have any use.